package.json

```diff
@@ -34,7 +34,7 @@
         "build-and-test": "npm run compile && npm run test",
         "test": "nyc mocha src/**/*.spec.ts --full-trace --bail",
         "coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
-        "postinstall": "node ./node_modules/vscode/bin/install",
+        "postinstall": "node .\node_modules/vscode/bin/install",
```

Comment:
This change doesn't look right. Those slashes mean the n is escaped, instead of being a directory separator.

Reply:
ugh - looks like a search and replace went astray. I'll fix it

Reply:
addressed